### PR TITLE
Add new adapter ServerBid (by Adzerk)

### PR DESCRIPTION
**Pull request by GaryKertis**
## Type of change

- [ ] Bugfix
- [ ] Feature
- [x] New bidder adapter
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Other

## Description of change
Add ServerBid (by Adzerk) adapter.

- test parameters for validating bids
```
{
            bidder: 'serverbid',
            params: {
                networkId: 9969,
                siteId: 730181,
            }
        }
```
- contact email gary@serverbid.com
- [x] official adapter submission

### CODE REVIEW

File: src/adapters/serverbid.js [hunk 1]
```diff
@@ -0,0 +1,166 @@
...
+
+  baseAdapter.callBids = function(params) {
+
+    if (params && params.bids && utils.isArray(params.bids) && params.bids.length) {
+
+      var data = {
+        placements: [],
+        time: Date.now(),
+        user: {},
+        url: document.location.href,
```
Comment (jaiminpanchal27) [inline src/adapters/serverbid.js]: I would recommend using these function utils.getTopWindowUrl() and utils.getTopWindowLocation()

File: src/adapters/serverbid.js [hunk 2]
```diff
@@ -0,0 +1,166 @@
...
+
+    for (var i = 0; i < bidIds.length; i++) {
+
+      bidId = bidIds[i];
+      bidObj = utils.getBidRequest(bidId);
+      bidCode = bidObj.bidder;
+      placementCode = bidObj.placementCode;
+
+      //prepare a no bid response.
+      bid = bidfactory.createBid(2, bidObj);
```
Comment (jaiminpanchal27) [inline src/adapters/serverbid.js]: Move this to else part. In case you get result, this code is of no use.

File: src/adapters/serverbid.js [hunk 3]
```diff
@@ -0,0 +1,166 @@
...
+        user: {},
+        url: document.location.href,
+        referrer: document.referrer,
+        enableBotFiltering: true,
+        includePricingData: true
+      };
+
+      var bids = params.bids || [];
+      for (var i = 0; i < bids.length; i++) {
+
```
Comment (jaiminpanchal27) [inline src/adapters/serverbid.js]: remove extra line breaks in this file.

File: src/adapters/serverbid.js [hunk 4]
```diff
@@ -0,0 +1,166 @@
...
+import bidfactory from 'src/bidfactory';
+import bidmanager from 'src/bidmanager';
+import * as utils from 'src/utils';
+import { ajax } from 'src/ajax';
+
+var ServerBidAdapter = function ServerBidAdapter() {
+
+  var baseAdapter = Adapter.createNew('serverbid');
+
+  var BASE_URI = '//e.serverbid.com/api/v2';
```
Comment (jaiminpanchal27) [inline src/adapters/serverbid.js]: Using const/let is recommended.

**Comment by GaryKertis** [comment]
@jaiminpanchal27 I have incorporated these changes. Please let me know if there are any other concerns. Thanks!

**Comment by jaiminpanchal27** [comment]
@GaryKertis utils.getTopWindowUrl is a function. Check line 54 in serverbid.js

**Comment by GaryKertis** [comment]
Oops. @jaiminpanchal27 fixed.

**Comment by jaiminpanchal27** [comment]
LGTM. Thanks @GaryKertis

**Comment by jaiminpanchal27** [comment]
This is merged into master. Please submit a PR to the [docs repo](https://github.com/prebid/prebid.github.io) to add a file for your adapter to the [bidders directory](https://github.com/prebid/prebid.github.io/tree/master/dev-docs/bidders) so your adapter's params will appear on the [bidders page](http://prebid.org/dev-docs/bidders.html). Thank you for contributing

**Pull request merged**